Formula/raylib.rb

```diff
@@ -0,0 +1,38 @@
+class Raylib < Formula
+  desc "Simple and easy-to-use library to learn videogames programming"
+  homepage "http://www.raylib.com/"
+  url "https://github.com/raysan5/raylib/archive/1.9.1-dev.tar.gz"
+  version "1.9.1-dev"
+  sha256 "892357fb44d340eb7449c23c425d660d98b34b91434400e7610514ef02698600"
+
+  head "https://github.com/raysan5/raylib.git", :branch => "develop"
+
+  option "with-pic", "Compile static library as position-independent code"
```

Comment:
why do we need this?

Reply:
You're writing a shared library that wraps raylib for some scripting language. Then you need it  as PIC to link it into the dylib, but normally you don't, so no need to generate less-efficient code unless the user asks for.

Reply:
In such a circumstance, I'm sure the user will be just fine compiling this on their own.

Reply:
I'd have liked the option, but ye I do compile it as my own. Removed.